<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShakemapImporterDialogBase</class>
 <widget class="QDialog" name="ShakemapImporterDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InaSAFE - Shakemap Importer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="main_stacked_widget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QWebView" name="help_web_view">
         <property name="url">
          <url>
           <string>about:blank</string>
          </url>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_1">
      <layout class="QGridLayout" name="gridLayout_6">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QWebView" name="info_web_view">
         <property name="url">
          <url>
           <string>about:blank</string>
          </url>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Input</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="2" column="0">
           <widget class="QLabel" name="label_title">
            <property name="text">
             <string>Title</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Input File (grid.xml)</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_source">
            <property name="text">
             <string>Source</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLineEdit" name="line_edit_title"/>
          </item>
          <item row="5" column="0">
           <widget class="QLineEdit" name="line_edit_source"/>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="input_path">
              <property name="enabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="open_input_tool">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
         <zorder>line_edit_title</zorder>
         <zorder>label_title</zorder>
         <zorder>line_edit_source</zorder>
         <zorder>label</zorder>
         <zorder>label_source</zorder>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Output</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="output_path">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="open_output_tool">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>The output will be a .tif raster file</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="use_output_default">
            <property name="text">
             <string>Same directory as input file</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="load_result">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Add output layer to QGIS project</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="keyword_wizard_checkbox">
            <property name="text">
             <string>Launch keywords wizard</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Algorithm</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QRadioButton" name="nearest_mode">
            <property name="text">
             <string>Nearest neighbour</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="inverse_distance_mode">
            <property name="text">
             <string>Inverse Distance</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>ShakemapImporterDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>ShakemapImporterDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_output_default</sender>
   <signal>toggled(bool)</signal>
   <receiver>output_path</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_output_default</sender>
   <signal>toggled(bool)</signal>
   <receiver>open_output_tool</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
